server/services/socket.js

```diff
@@ -29,9 +37,13 @@ exports.closeSocket = () => {
 exports.isCurrentPlayerSocket = (socket) => gameLogic.getGameState().currentPlayer && socket.handshake.session.user === gameLogic.getGameState().currentPlayer.username;
 
 // Emit all the players
-const emitPlayers = players => io.emit("players", { players });
+const emitPlayers = (players) => io.emit("players", { players });
```

Comment:
Didn't need to add these brackets

Reply:
Just keeping consistent with the style of the rest of the file

Reply:
I'll remove all brackets for the single parameters